site-builder/src/main.rs

```diff
@@ -212,8 +212,22 @@ async fn run() -> Result<()> {
 
     let args = Args::parse();
     let mut config: Config = std::fs::read_to_string(&args.config)
+        .or_else(|err| {
+            if err.kind() == std::io::ErrorKind::NotFound {
+                let config;
+                if let Some(home_dir) = home::home_dir() {
+                    config =
+                        std::fs::read_to_string(home_dir.join(".walrus").join("sites-config.yaml"));
```

Comment:
I think, the CLI checks for the Walrus configuration file in the following order:

1. If a path is specified via the CLI option, it will attempt to read the config file from the given path. An error is returned if the path is invalid.
2. If no path is provided explicitly, the CLI will search for `client_config.yaml` or `client_config.yml` in the following locations:
   - The current working directory (`./`).
   - If the environment variable `XDG_CONFIG_HOME` is set, it will search in `$XDG_CONFIG_HOME/walrus/`.
   - If not, it will check in `~/.config/walrus/`.
   - Finally, it will check `~/.walrus/`.

I think that, as with client_config.yaml, it might be better to follow a similar approach here.

Reply:
Thank you @chariskms ! I will change this asap 😄

Reply:
Done! Uploaded and tested ✅ 
Everything works as expected.

Reply:
For future reference of @chariskms comment: https://github.com/MystenLabs/walrus/blob/main/crates/walrus-service/src/client/cli/args.rs#L23